ui/src/app/views/concept-set-details/component.html

```diff
@@ -1,11 +1,25 @@
 <app-top-box>
   <div class="concept-set-header">
-    <!-- TODO(calbach): Implement the action menu. -->
     <clr-dropdown class="action-menu-dropdown">
-      <button type="button" class="resource-menu" [disabled]="true" clrDropdownTrigger>
+      <button type="button" class="resource-menu" [disabled]="!canEdit" clrDropdownTrigger>
         <clr-icon shape="ellipsis-vertical" class="ellipsis-vertical"></clr-icon>
       </button>
-      <clr-dropdown-menu *clrIfOpen></clr-dropdown-menu>
+      <clr-dropdown-menu *clrIfOpen>
+        <button clrDropdownItem class="action-item action-edit"
+            (click)="editing=true"
+            [class.disabled]="!canEdit"
+            [disabled]="!canEdit">
+          <clr-icon shape="pencil"></clr-icon>
+          Edit
+        </button>
+        <button clrDropdownItem class="action-item action-delete"
+            (click)="deleteModal.open()"
+            [class.disabled]="!canEdit"
```

Comment:
Since the resource menu will already be disabled, i am not sure if we need [disabled] options for action items

Reply:
Added a comment - the disabled dropdown is temporary and we'll permanently enable it once we have clone, so I'm just preemptively disabling the buttons as well - does this sound OK?

Reply:
ah ok